cli/command/secret/create.go

```diff
@@ -45,7 +45,7 @@ func newSecretCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.StringVarP(&options.driver, "driver", "d", "", "Secret driver")
 	flags.SetAnnotation("driver", "version", []string{"1.31"})
 	flags.StringVar(&options.templateDriver, "template-driver", "", "Template driver")
-	flags.SetAnnotation("driver", "version", []string{"1.37"})
+	flags.SetAnnotation("template-driver", "version", []string{"1.37"})
```

Comment:
ahum (introduced in https://github.com/docker/cli/pull/896) 😊; looks like we fixed this for `config` but not `secrets` https://github.com/docker/cli/pull/1769

Could you open a separate PR for this change so that we can cherry-pick this

Reply:
Sure, and sorry that slipped in, I will check my diff better next time :)

Reply:
no, it's good that you found it (thanks!) just if it's a separate PR, I can cherry-pick if 😄

Reply:
Thanks for spotting that; I opened #1785.